tests/unit/test_run.py

```diff
@@ -1218,47 +1212,36 @@ def test_append_duplicate_times(test_append_scm_runs, duplicate_msg):
     )
 
 
-def test_append_doesnt_warn_if_continuous_times(test_append_scm_dfs):
```

Comment:
why removed?

Reply:
The test wasn't using ScmRun, rather ScmDataFrame so was failing. The test and functionality was added back in 
#110  https://github.com/openscm/scmdata/pull/110/files#diff-af66f4e650e54b5f9a125a48a0695accR1301